### ISSUE TITLE
Unused parameter Chi

### ISSUE THREAD

**Issue by omershlo** [ligeroinc/LigeroRSA#4]
https://github.com/ligeroinc/LigeroRSA/blob/master/include/LatticeEncryption.hpp#L500 Looks like Chi is not used in this function. Consider removing it.

**Comment by tarikriviere** [comment]
Indeed; this was just correct [commit: 20570edb7deaa83212f7c026f533f189d66f2e39]. Thanks.
